src/coinbase/wallet.ts

```diff
@@ -130,31 +136,82 @@ export class Wallet {
   }
 
   /**
-   * Imports a Wallet for the given Wallet data.
+   * Imports an external wallet into CDP using a BIP-39 mnemonic seed phrase.
    *
-   * @param data - The Wallet data to import.
-   * @param data.walletId - The ID of the Wallet to import.
-   * @param data.seed - The seed to use for the Wallet.
+   * @param mnemonicSeedPhrase - The BIP-39 mnemonic seed phrase (12, 15, 18, 21, or 24 words).
    * @returns The imported Wallet.
-   * @throws {ArgumentError} If the Wallet ID is not provided.
-   * @throws {ArgumentError} If the seed is not provided.
+   * @throws {ArgumentError} If the seed phrase is not provided or invalid.
+   * @throws {ArgumentError} If the seed phrase is not 12, 15, 18, 21, or 24 words.
    * @throws {APIError} If the request fails.
    */
-  public static async import(data: WalletData): Promise<Wallet> {
-    if (!data.walletId) {
+  public static async importFromMnemonicSeedPhrase(mnemonicSeedPhrase: string): Promise<Wallet> {
```

Comment:
`importFromMnemonicSeedPhrase` is quite verbose. The PRD specifies `import` method should take mnemonic seed as input. 

Did we discuss with product the deprecation and method name changes?

Reply:
Resolved via offline convo. Changes appear in latest commit